srv/components/ha/cortx-ha/config.sls

```diff
@@ -15,23 +15,17 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-include:
-  - components.ha.cortx-ha.install
 
 Run cortx-ha post_install:
   cmd.run:
     - name: __slot__:salt:setup_conf.conf_cmd('/opt/seagate/cortx/ha/conf/setup.yaml', 'ha:post_install')
-    - require:
```

Comment:
Let's maintain these `require` statements to ensure installation.
Any specific reason this is removed here?

Reply:
We don't want to apply installation stages during configuration phases in scope of rollback (and it's not the only case, see below). Just few components have that logic and it is not consistent. We need to choose only one approach for all components:

- either call stages (prepare, install, config ....) separately and manage the flow ourselves (e.g. in init.sls or in api logic)
- or let salt to relate all stages so we can apply only a final one.

Keeping in mind the new workflow of the development I think the first one is more applicable for us.

Reply:
Got it.
But then it has to be removed from all other component configs as well.